docs/communication.md

```diff
@@ -0,0 +1,34 @@
+# Communication
+
+## Custom messages
+
+Luigi Core and Luigi Client can exchange custom messages in two possible ways.
+
+#### Luigi Client sends a custom message to Luigi Core
+
+For Luigi Client, you need to use the [*sendCustomMessage*](luigi-client-api.md#sendCustomMessage) method from Client API.
+For Luigi Core, you need to set something similar to the following on the root level of your Luigi configuration object:
```

Comment:
```suggestion
For Luigi Core to receive custom messages, define a configuration similar to the following at the root level of your Luigi configuration object:
```

Reply:
I will apply your suggestion, but I will use "process" instead of "receive", since the message actually always arrives to Core.

Reply:
Sounds good to me :)